src/fmu/tools/sensitivities/_excel2dict.py

```diff
@@ -84,8 +84,9 @@ def _find_geninput_sheetname(input_filename):
 
     if len(general_input_sheet) > 1:
         raise ValueError(
-            "More than one sheet with general input"
-            "Sheetnames are {} ".format(general_input_sheet)
+            "More than one sheet with general input. Sheetnames are {} ".format(
+                general_input_sheet
+            )
```

Comment:
You might consider making these regular format strings,

`f"More than one sheet with general input. Sheetnames are {general_input_sheet} "`

I'm surprised ruff doesn't catch it. Maybe there's a reason for the method call?

Reply:
or maybe be specific and add the `UP032` https://docs.astral.sh/ruff/rules/f-string/ ?

Reply:
I'm not sure we can completely use `UP` because sometimes we still need to fallback on Python 3.8 type annotations, even with importing future annotations. It might be reasonable to add `UP` completely and exclude this (or other) particular rules.

Reply:
Either way, maybe that should be a separate PR, so we can keep the configurations in-sync across all repositories 😄

Reply:
Ok, sounds reasonable with separate PR 🙂  then I'll merge this now!